### PR TITLE
add debian 10 image

### PR DESCRIPTION
**Pull request by nonifier**

### CODE REVIEW

**Comment by xlqian** [comment]
with a new PR?

File: .github/workflows/docker_image.yml [hunk 1]
```diff
@@ -2,18 +2,40 @@ name: Docker Image CI
...
-    types:
-    - created 
+  push:
+    branches:
+    - master
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
+    info:
```
Comment (benoit-bst) [inline .github/workflows/docker_image.yml]: it is used for what ?

Comment (mbounabi) [reply .github/workflows/docker_image.yml]: +1

Comment (benoit-bst) [reply .github/workflows/docker_image.yml]: As he said, it it for test purposes

**Comment by nonifier** [comment]
@xlqian You have 2 ways to trigger the build:
```yaml
on: 
  pull_request: # 1
  push:            # 2
    branches:
    - master
```
#1 : when a pull request is opened, or synced (repushed).
#2 : when something is pushed to the master branch (eg. a PR is merged to master)

You can have more info here : https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow

**Pull request merged**